src/goto-programs/abstract-interpretation/gcse.cpp

```diff
@@ -0,0 +1,468 @@
+#include "goto-programs/goto_program.h"
+#include "irep2/irep2_expr.h"
+#include "irep2/irep2_type.h"
+#include "pointer-analysis/value_set_analysis.h"
+#include "util/std_code.h"
+#include "util/std_expr.h"
```

Comment:
How about including all these via the `<...>` form? The `"..."` one can't find them anyway in the search path from starting from this file.